integration-tests/api-gateway/env-variables.ts

```diff
@@ -13,22 +11,15 @@ export const CLIENT_ASSERTION_TYPE =
   "urn:ietf:params:oauth:client-assertion-type:jwt-bearer";
 export const GRANT_TYPE = "authorization_code";
 export const environment = process.env.Environment || "localdev";
-export const getClaimSet = async () => {
+export const testResourcesStack = process.env.Environment || "test-resources";
```

Comment:
Hmm is this right? would we expect the env var `Environment` to be set to a test resources stack?